certsuite/harness.py

```diff
@@ -382,19 +380,19 @@ def list_tests(args, config):
         print "%s:%s" % (test, group)
     return True
 
-def edit_device_profile(device_profile_path):
+def edit_device_profile(device_profile_path, message):
     resp = ''
+    result = None
     try:
-        env = environment.get(environment.InProcessTestEnvironment, addr=None, verbose=False)
+        env = environment.get(environment.InProcessTestEnvironment)
         url = "http://%s:%d/profile.html" % (env.server.addr[0], env.server.addr[1])
         webbrowser.open(url)
         environment.env.handler = server.wait_for_client()
-        
-        question = ''
-        resp = environment.env.handler.prompt(question)
 
-        message = 'Create device profile failed!!'    
-        if resp != '':
+        resp = environment.env.handler.prompt(message)
+
+        message = 'Create device profile failed!!'
+        if resp is not None and resp != '':
```

Comment:
err... That would be my fault if I didn't make it clear. I apologize for asking refine twice. None and empty string are all falsy statement. Therefore, the following code should work:
```python
if not resp:
```